xExtension-YouTube/configure.phtml

```diff
@@ -1,38 +1,41 @@
-
-<form action="<?php echo _url('extension', 'configure', 'e', urlencode($this->getName())); ?>" method="post">
-	<input type="hidden" name="_csrf" value="<?php echo FreshRSS_Auth::csrfToken(); ?>" />
+<form action="<?= _url('extension', 'configure', 'e', urlencode($this->getName())) ?>" method="post">
+	<input type="hidden" name="_csrf" value="<?= FreshRSS_Auth::csrfToken() ?>" />
 	<div class="form-group">
 
-		<label class="group-name" for="yt_height"><?php echo _t('ext.yt_videos.height'); ?></label>
+		<label class="group-name" for="yt_height"><?= _t('ext.yt_videos.height') ?></label>
 		<div class="group-controls">
-			<input type="number" id="yt_height" name="yt_height" value="<?php echo FreshRSS_Context::$user_conf->yt_player_height; ?>" min="50" data-leave-validation="1">
+			<input type="number" id="yt_height" name="yt_height" value="<?= $this->getHeight() ?>" min="50" data-leave-validation="1">
```

Comment:
@marienfressinaud do you think this is a safe decision to use `this` here for the future? Feels like "works by accident" or was this your intention originally when adding plugin support?

Reply:
Can't remember what was my first intention, but at least I used it in the [CustomCSS extension](https://github.com/FreshRSS/Extensions/blob/master/xExtension-CustomCSS/configure.phtml). So if it wasn't officially supported, let's say it is now :D

Reply:
Thats a word 😁 resolved